src/plugins/saved_search/public/services/saved_searches/saved_search_attribute_service.test.ts

```diff
@@ -1,250 +0,0 @@
-  const actual = jest.requireActual('./create_get_saved_search_deps');
-  let deps: GetSavedSearchDependencies;
-  return {
-    ...actual,
-    createGetSavedSearchDeps: jest.fn().mockImplementation((services) => {
-      if (deps) return deps;
-      deps = actual.createGetSavedSearchDeps(services);
-      return deps;
-    }),
-  };
-});
-
-jest
-  .spyOn(mockServices.contentManagement, 'update')
-  .mockImplementation(async ({ id }) => ({ item: { id } }));
-
-jest.spyOn(mockServices.contentManagement, 'get').mockImplementation(async ({ id }) => ({
-  item: { attributes: { id }, references: [] },
-  meta: { outcome: 'success' },
-}));
-
-describe('getSavedSearchAttributeService', () => {
-  it('should return saved search attribute service', () => {
-    const savedSearchAttributeService = getSavedSearchAttributeService(mockServices);
-    expect(savedSearchAttributeService).toBeDefined();
-  });
-
-  it('should call saveSearchSavedObject when wrapAttributes is called with a by ref saved search', async () => {
-    const savedSearchAttributeService = getSavedSearchAttributeService(mockServices);
-    const savedObjectId = 'saved-object-id';
-    const input: SearchByReferenceInput = {
-      id: 'mock-embeddable-id',
-      savedObjectId,
-      timeRange: { from: 'now-15m', to: 'now' },
-    };
-    const attrs: SavedSearchByValueAttributes = {
-      title: 'saved-search-title',
-      sort: [],
-      columns: [],
-      grid: {},
-      hideChart: false,
-      isTextBasedQuery: false,
-      kibanaSavedObjectMeta: {
-        searchSourceJSON: '{}',
-      },
-      references: [],
-    };
-    const result = await savedSearchAttributeService.wrapAttributes(attrs, true, input);
-    expect(result).toEqual(input);
-    expect(saveSearchSavedObject).toHaveBeenCalledTimes(1);
-    expect(saveSearchSavedObject).toHaveBeenCalledWith(
-      savedObjectId,
-      {
-        ...omit(attrs, 'references'),
-        description: '',
-      },
-      [],
-      mockServices.contentManagement
-    );
-  });
-
-  it('should call getSearchSavedObject when unwrapAttributes is called with a by ref saved search', async () => {
-    const savedSearchAttributeService = getSavedSearchAttributeService(mockServices);
-    const savedObjectId = 'saved-object-id';
-    const input: SearchByReferenceInput = {
-      id: 'mock-embeddable-id',
-      savedObjectId,
-      timeRange: { from: 'now-15m', to: 'now' },
-    };
-    const result = await savedSearchAttributeService.unwrapAttributes(input);
-    expect(result).toEqual({
-      attributes: {
-        id: savedObjectId,
-        references: [],
-      },
-      metaInfo: {
-        sharingSavedObjectProps: {
-          outcome: 'success',
-        },
-      },
-    });
-    expect(getSearchSavedObject).toHaveBeenCalledTimes(1);
-    expect(getSearchSavedObject).toHaveBeenCalledWith(
-      savedObjectId,
-      createGetSavedSearchDeps(mockServices)
-    );
-  });
-
-  describe('toSavedSearch', () => {
```

Comment:
Do we still have this test somewhere?

Reply:
Re-added this test in 66cd738e931cf2d34c4ff0ce9fa4aa78b219b448 - thanks for catching that!